luna/pretrained_models/googlenet.py

```diff
@@ -11,8 +11,8 @@
 from tensorflow.keras.layers import Layer, Input, Dense, Conv2D, MaxPooling2D
 from tensorflow.keras.layers import AveragePooling2D, ZeroPadding2D, Dropout, Flatten
 from tensorflow.keras.layers import Concatenate, Activation
+from tensorflow.python.keras.utils import conv_utils
```

Comment:
We are using convert_kernel directly below. We do not import it anymore. How does that work?

Reply:
done